src/scriptworker/cot/verify.py

```diff
@@ -1165,6 +1165,11 @@ async def _get_additional_github_push_jsone_context(decision_link):
     if committer_login == "web-flow":
         committer_login = commit_data["author"]["login"]
 
+    # This value could have been taken from `commit_data.parents[0]` too but
+    # it is more visible if picked up from `.taskcluster.yml` env vars
+    base_prefix = "{}_BASE_REV".format(context.config["source_env_prefix"])
+    before_hash = task["payload"]["env"][base_prefix]
```

Comment:
If I don't change the mpd test file, and I change this to a `.get()`, it works.

Then if I `verify_cot --cot-product mobile --no-check-task --task-type scriptworker --cleanup -- SyfDmaLsR6utrjkL5FmbhA` , that succeeds.